### PR TITLE
Add bxt_force_zmax

### LINKED ISSUE #178: bxt_force_zmax <value>
Forces `sv_zmax` to keep value after saveload (cuz its resets to **4096** by default)

Got that enhancement from: https://github.com/sharkov63/SaveWarpTool

### PR DESCRIPTION
**Pull request by hobokenn**
closes #178

### CODE REVIEW

File: BunnymodXT/modules/HwDLL.cpp [hunk 1]
```diff
@@ -4568,3 +4612,15 @@ HOOK_DEF_2(HwDLL, void, __cdecl, SV_WriteEntitiesToClient, client_t*, client, vo
 	if (CVars::_bxt_norefresh.GetBool())
 		*num_edicts = orig_num_edicts;
 }
+
+HOOK_DEF_0(HwDLL, void, __cdecl, SV_SetMoveVars)
+{
+	ORIG_SV_SetMoveVars();
+
+	if (CVars::sv_cheats.GetBool() && CVars::bxt_force_zmax.GetBool()) {
```
Comment (SmileyAG) [inline BunnymodXT/modules/HwDLL.cpp:4620]: Any reason to make this cvar was cheats-protected?

**Review by YaLTeR — APPROVED** [review]

File: BunnymodXT/modules/HwDLL.cpp (hunk 1 above)
Comment (SmileyAG) [reply BunnymodXT/modules/HwDLL.cpp:4620]: `sv_zmax` works without any cheats, thats why I'm talking

Comment (hobokenn) [reply BunnymodXT/modules/HwDLL.cpp:4620]: @YaLTeR what do you think about ^

Comment (YaLTeR) [reply BunnymodXT/modules/HwDLL.cpp]: ¯\_ (ツ)_/¯

**Comment by YaLTeR** [comment]
Could you resolve the conflicts please?

**Comment by YaLTeR** [comment]
Thanks!

**Pull request merged**